README.md

```diff
@@ -92,15 +91,15 @@ Which one of the following concepts does this best illustrate?
 [ ] A: Modularity
 [ ] B: MVC
 [ ] C: Two-way data-binding
```

Comment:
Correct answer is two way data binding. Changes in the view are immediately reflected in the model, and vice-versa.